cmd/edgectl/main.go

```diff
@@ -183,7 +183,7 @@ func getRootCommand() *cobra.Command {
 	}
 	loginCmd := &cobra.Command{
 		Use:   "login [flags] HOSTNAME",
-		Short: "Access the Ambassador Edge Policy Console",
+		Short: "Log in and open the Ambassador Edge Policy Console web application",
```

Comment:
The [marketing-approved documentation](https://www.getambassador.io/about/edge-policy-console/) says 

> After you install the Ambassador Edge Stack, you can log in to the Edge Policy Console (EPC) to manage your deployment.

Drawing on that phrasing, I suggest "Log in to the Ambassador Edge Policy Console"

Reply:
In particular, I don't think we should use the phrase "web application" to describe the console. Yes, I know that does describe it correctly, but our documentation consistently uses that phrase to describe a service that a user writes and then exposes using Ambassador.

Reply:
The point of the word "open" is that it describes the fact that it will open in the web browser; logging in is just the beginning of the process.

Removing the "web application" text sounds right given the need to stay consistent with the documentation.

Reply:
@brucehorn did you decide to abandon the "and open" as well (which I thought was an improvement)?

Reply:
I thought it was an improvement as well but deferred to @ark3